### PR TITLE
Fix null pointer exception during the generation queries which return…

### PR DESCRIPTION
**Pull request by dkaera**
Recreated PR because the previous one was mistakenly updated during the solving conflicts.

### CODE REVIEW

File: example/pubspec.lock [hunk 1]
```diff
@@ -505,14 +505,14 @@ packages:
       name: web_socket_channel
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.0"
+    version: "2.2.0"
```
Comment (mqus) [inline example/pubspec.lock:508]: Nit: Maybe separate the updated pubspec into another commit next time, since it isn't really necessary for this change. But this is really a minor issue.

**Review by mqus — APPROVED** [review]

File: example/pubspec.lock (hunk 1 above)
Comment (dkaera) [reply example/pubspec.lock:508]: agree

To be honest, I think `pubspec.lock` should be added to `gitignore` since it is a generated pub file. 

Yes, the dart documentation also [mentions this](https://dart.dev/guides/libraries/private-files#the-rules). So we can remove it from the example project but must leave it in packages.

Comment (mqus) [reply example/pubspec.lock:508]:
```
pubspec.lock  # Except for application packages
```
I interpret this to mean that it should be .gitignore'd for libraries but not for applications. Technically, the example is an application, not a library.

Comment (dkaera) [reply example/pubspec.lock:508]: ah, yeah, you're right 😂

**Review by ThomasMiddel — APPROVED** [review]

**Review by ThomasMiddel — APPROVED** [review]

**Review by memoriasIT — APPROVED** [review]
